core/DCache.hpp

```diff
@@ -26,29 +28,77 @@ namespace olympia
             PARAMETER(uint32_t, l1_associativity, 8, "DL1 associativity (power of 2)")
             PARAMETER(uint32_t, cache_latency, 1, "Assumed latency of the memory system")
             PARAMETER(bool, l1_always_hit, false, "DL1 will always hit")
+            PARAMETER(uint32_t, mshr_entries, 8, "Number of MSHR Entries")
         };
 
         static const char name[];
         DCache(sparta::TreeNode* n, const CacheParameterSet* p);
 
       private:
+        ////////////////////////////////////////////////////////////////////////////////
+        // L1 Cache Handling
```

Comment:
Since this is DCache, would you change all references to L1 Cache to DCache unless you already have some other plans for ICache/IL1Cache.